tests/ft_cs.vader

```diff
@@ -12,7 +12,19 @@ Execute (cs: msbuild: errorformat):
     \ 'col': 63,
     \ 'valid': 1,
     \ 'vcol': 0,
-    \ 'nr': -1,
+    \ 'nr': 1002,
     \ 'type': 'E',
     \ 'pattern': '',
-    \ 'text': 'error CS1002: ; expected [Foo\Foobar.csproj]'}]
+    \ 'text': 'error CS1002: ; expected'}]
+
+  lgetexpr 'Program.cs(25,29): warning CS0168: The variable 'ex' is declared but never used [Foo\Foobar.csproj]'
+  AssertEqual getloclist(0), [{
+    \ 'lnum': 25,
+    \ 'bufnr': bufnr('%'),
+    \ 'col': 29,
+    \ 'valid': 1,
+    \ 'vcol': 0,
+    \ 'nr': 168,
+    \ 'type': 'W',
+    \ 'pattern': '',
+    \ 'text': 'The variable 'ex' is declared but never used'}]
```

Comment:
The test fails:

> [{'lnum': 21, 'bufnr': 3, 'col': 63, 'valid': 1, 'vcol': 0, 'nr': 1002, 'type': 'E', 'pattern': '', 'text': 'error CS1002: ; expected'}] should be equal to [{'lnum': 21, 'bufnr': 3, 'col': 63, 'valid': 1, 'vcol': 0, 'nr': 1002, 'type': 'E', 'pattern': '^\VFoo\Foobar.csproj\$', 'text': '; expected'}]

Likely just because you got the quoting wrong (not escaped).. both in `lgetexpr` and `'text'`.